scss/forms/_form-check.scss

```diff
@@ -87,11 +88,9 @@
 
     @if $enable-gradients {
       background-image: escape-svg($form-check-input-indeterminate-bg-image), var(--#{$prefix}gradient);
-      background-position: 50% add(50%, $border-width * .25), center; // Boosted mod
```

Comment:
Without `background-position` in both cases, I observe a different rendering on Firefox and Chrome. Shouldn't we keep it and change the value so that it is perfectly centered?